test/integration/targets/win_format/tasks/tests.yml

```diff
@@ -136,3 +139,24 @@
       - format_volume_without_force is failed
       - 'format_volume_without_force.msg == "Force format must be specified to format non-pristine volumes"'
       - format_volume_with_force is changed
+
+- name: Reformat using different alu without force format
+  win_format:
+    path: "{{ shell_partition_result.stdout | trim }}"
+    allocation_unit_size: 8192
+    file_system: ntfs
+  register: reformat_using_alu_without_force
+  ignore_errors: True
+
+- name: Reformat using different alu using force format
+  win_format:
+    path: "{{ shell_partition_result.stdout | trim }}"
+    allocation_unit_size: 8192
+    file_system: ntfs
+    force: True
+  register: reformat_using_alu_with_force
+
+- assert:
```

Comment:
There is no direct logical connection between the two tests so maybe assert each individually